src/main/resources/logback-spring.xml

```diff
@@ -6,7 +6,33 @@
 
     <springProperty scope="context" name="app" source="spring.application.name"/>
 
-    <logger name="uk.gov.justice.digital.hmpps.personrecord" additivity="false" level="DEBUG">
+    <appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>${LOG_PATTERN}</pattern>
+        </encoder>
+    </appender>
+
+    <logger name="uk.gov.justice.digital.hmpps.personrecord.HmppsPersonRecordKt" additivity="false" level="DEBUG">
+        <appender-ref ref="consoleAppender"/>
+    </logger>
+
+    <logger name="uk.gov.justice.digital.hmpps" additivity="false" level="DEBUG">
+        <appender-ref ref="consoleAppender"/>
+    </logger>
+
+    <logger name="org.springframework" additivity="false" level="INFO">
+        <appender-ref ref="consoleAppender"/>
+    </logger>
+
+    <logger name="com.microsoft.applicationinsights" additivity="false" level="INFO">
+        <appender-ref ref="consoleAppender"/>
+    </logger>
+
+    <logger name="org.apache.catalina" additivity="false" level="INFO">
```

Comment:
I would go with WARN for most of these - do we really care?

Reply:
Just set to INFO for all of them as most of the noisy logs are set to DEBUG